src/tty-ask-password-agent/tty-ask-password-agent.c

```diff
@@ -155,13 +154,17 @@ static int agent_ask_password_tty(
                 if (r < 0)
                         log_warning_errno(r, "Failed to reset terminal, ignoring: %m");
 
+                log_info("Starting password query on %s", con);
```

Comment:
End sentence with a full stop?

Reply:
I thought that /dev/something. might be a bit confusing, but OK.